app/enums/branch.ts

```diff
@@ -0,0 +1,27 @@
+export enum Branch {
+  Main = "main",
+  JeleniaGora = "jelenia_gora",
+  Walbrzych = "walbrzych",
+  Legnica = "legnica",
+}
+
+export const branchEnumValues = {
+  availableValues: [
+    {
+      value: Branch.Main,
+      label: "Main branch",
+    },
```

Comment:
"Main campus" would probably be a better description here.
this whole `branchEnumValues` constant likely won't be used for much longer anyway, as we're trying to get rid of the adminjs panel